vision/classification/classification_utils.go

```diff
@@ -18,7 +18,7 @@ func Overlay(img image.Image, classifications Classifications) (image.Image, err
 	y := 30
 	for _, classification := range classifications {
 		rimage.DrawString(gimg,
-			fmt.Sprintf("%v: %.2f %%", classification.Label(), classification.Score()*100.),
+			fmt.Sprintf("%v: %.2f %%", classification.Label(), classification.Score()),
```

Comment:
remove the percentage sign in the string as well

Reply:
done. Still not sure why would someone get a confidence bigger than 1 after a "softmax" activation.